DropboxLike.Api/Program.cs

```diff
@@ -15,13 +16,17 @@
 builder.Services.Configure<AwsConfiguration>(options =>
 {
     options.BucketName = builder.Configuration.GetSection("Aws:BucketName").Get<string>() ?? string.Empty;
+    // options.BucketName = builder.Configuration.GetSection("Aws:AWS_ACCESS_KEY").Get<string>() ?? string.Empty;
```

Comment:
We should not leave comments in code that should be self-documenting. Please remove them.